assets/css/_cutout_overlay.scss

```diff
@@ -0,0 +1,44 @@
+.c-overlay-cutout {
```

Comment:
Not sold on the name of this component tbh

Reply:
It looks like the file is called `_cutout_overlay.scss` whereas the class is `c-overlay-cutout`. I like the former better, since it emphasizes that "cutout" is modifying "overlay," which I think is the correct way of thinking about it (it's an overlay, with a cutout). Maybe including the word "map" somewhere in the name would also help indicate what it's for.

Reply:
ah I forgot to update this when I updated the file name, thanks

Reply:
I feel like in a case like this we developers are going to be the primary people interacting with the component name I'm okay with prioritizing making it clearer to devs.